examples/golden_graph/node.hpp

```diff
@@ -11,6 +11,10 @@ class Node
 	Node(int id, int value);
 public:
 	void deleteThis();	
+	int getID() const;
+	int getValue() const;
+	const std::vector<Edges*>* getEdges() const;
```

Comment:
change std::vector<Edges*>*  to  std::vector<Edges*>**&**

Reply:
Thank you for your attention, I will remove the defect.